.env.example

```diff
@@ -15,3 +15,10 @@ SENTRY_DNS=
 
 # Log level to use, DEBUG by default in dev
 LOG_LEVEL=DEBUG
+
+# Postgres database
+POSTGRES_DB_NAME=open_prices
```

Comment:
We can keep it simple locally (postgres/postgres/postgres for DB name, user and password), as we use Docker usually and it's for local dev only